qutebrowser/browser/browsertab.py

```diff
@@ -826,6 +833,19 @@ def networkaccessmanager(self) -> typing.Optional[QNetworkAccessManager]:
     def shutdown(self) -> None:
         raise NotImplementedError
 
+    def run_js_sync(
+            self,
+            code: str, *,
+            world: typing.Union[usertypes.JsWorld, int] = None
```

Comment:
Since it's only implemented for QtWebKit which can't handle the `world` argument, no need to have it here.